lib/php/lib/Factory/TFramedTransportFactory.php

```diff
@@ -1,13 +1,32 @@
 <?php
 
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
 namespace Thrift\Factory;
 
 use Thrift\Transport\TFramedTransport;
 use Thrift\Transport\TTransport;
 
 class TFramedTransportFactory implements TTransportFactoryInterface
 {
-    public static function getTransport(TTransport $transport)
```

Comment:
instances of TTransportFactoryInterface is always created through new object so there is no any requirement to use here static method.

Discovered this when writing test to TSimpleServer